x/executionlayer/genesis.go

```diff
@@ -40,8 +41,19 @@ func InitGenesis(
 		keeper.SetGenesisAccounts(ctx, data.Accounts)
 	}
 	keeper.SetChainName(ctx, data.ChainName)
+	stateHash, bonds, errStr := grpc.Commit(keeper.client, util.DecodeHexString(util.StrEmptyStateHash), response.GetSuccess().GetEffect().GetTransformMap(), genesisConfig.GetProtocolVersion())
```

Comment:
Would not the parentState arg of the Commit call be the genesis commit returned from RunGenesis?

Reply:
Do the same, but to use the bonds applied to the ee